### ISSUE TITLE
Linux Support

### ISSUE THREAD

**Issue by matt-c1** [yowipr/Manual#2]
Hi! I'm making this issue to track if and when Linux support will be added. I don't instantly see any game-breakers: C#, Comfy, and Stable Diffusion all work on Linux.

**Comment by yowipr** [comment]
the real breaker: my sanity
Manual is made in WPF C#, and this is for windows only. To support linux, I have to migrate the whole code to MAUI or something like that.
I was planning to do it for the next big update, if I can find someone who knows about linux and maui 😂

**Comment by hartmark** [comment]
Yeah, it's quite the effort to port it over to MAUI for example. I haven't worked with WPF in years and have no experience with MAUI, but I know a lot of Linux though.

**Comment by matt-c1** [comment]
Oh, I see. ".NET Multi-platform App UI" is not multi-platform at all, haha. So I guess this is a "won't fix" unless the author plans on switching to an actually multi-platform UI framework and perhaps just re-writing the entire program, or rather, making an entirely different program from scratch.

**Comment by RuggedPineapple** [comment]
There's a Senior Microsoft Engineer who has a maui-linux repo:  https://github.com/jsuarezruiz/maui-linux
